controllers/presentationController.js

```diff
@@ -21,21 +21,37 @@ async function getAllPresentations(req, res) {
   }
 }
 
-// async function createPresentation(req, res, next) {
-//   // create
-// }
+async function createPresentation(req, res) {
```

Comment:
```suggestion
exports.createPresentation = async (req, res) => {
  // ..
```
컨트롤러에 미들웨어 함수들을 익명 함수를 사용한 표현식으로 바꿔 이를 exports의 메소드로 구현하는 식으로 바꾸는 게 좋을 것 같습니다.